src/dns/aardvark.rs

```diff
@@ -265,14 +268,27 @@ impl Aardvark {
                                                 container_ip_v6 = container_ip.to_string();
                                             }
 
+                                            // unwrap is safe since we already have statusblock
+                                            // for this network
+                                            let mut name_vector =
+                                                Vec::from([container_name.clone()]);
+                                            if let Some(alias) = &per_network_opts
+                                                .get(&network_name)
+                                                .unwrap()
```

Comment:
Probably safer to `unwrap_or_else`

Reply:
@mheon We generate status block via per_network_opts for which are already doing a check. So its not likely that we will get `None` here, i also added a comment https://github.com/containers/netavark/pull/184/files#diff-6307ba066c1817023f2a53b2c7c6a64d35bde6cf7f443cf5b07f34b9f163513bR271

But I am okay if you still like a handling here.

Reply:
Seems safest, in case the rest of the code changes around us

Reply:
removed unwrap() in latest commit.